packages/constants/src/payload/payload.types.ts

```diff
@@ -96,6 +98,8 @@ export interface NFTResponsePayload {
   nfts: AccountNFToken[] | null | undefined;
 }
 
+export const DEFAULT_MEMO_TYPE = 'Description';
```

Comment:
This is a constant and not a type, I don't think it should live in this file.

Reply:
Done [here](https://github.com/GemWallet/gemwallet-extension/pull/144/commits/1e35ec5f0af03a43e69142316981a9a5399715bc)